cmd/servicelog/list.go

```diff
@@ -19,8 +19,12 @@ var listCmd = &cobra.Command{
 	Use:   "list",
```

Comment:
```suggestion
	Use:   "list [flags] [options] cluster-identifier",
```

Reply:
This change will change the help output from

```
$ osdctl servicelog list --help
gets all servicelog messages for a given cluster

Options:
  -A, --all-messages=false: Toggle if we should see all of the messages or only SRE-P specific ones

Usage:
  osdctl servicelog list [flags] [options]

Use "osdctl options" for a list of global command-line options (applies to all commands).
```

To
```
$ ./osdctl servicelog list --help
gets all servicelog messages for a given cluster

Options:
  -A, --all-messages=false: Toggle if we should see all of the messages or only SRE-P specific ones

Usage:
  osdctl servicelog list [flags] [options] cluster-id

Use "osdctl options" for a list of global command-line options (applies to all commands).
``` 

(note the "Usage" field